codegen.swift

```diff
@@ -81,44 +81,49 @@ public enum FontAwesome: String {
 
 let sortedKeys = Array(icons.keys).sorted(by: <)
 sortedKeys.forEach { key in
-    guard let value = icons[key] else { return }
+    guard icons[key] != nil else { return }
     let enumKeyName = key.filteredKeywords().camelCased(with: "-")
     fontAwesomeEnum += """
-        case \(enumKeyName) = \"\\u{\(value.unicode)}\"
+        case \(enumKeyName) = \"fa-\(key)\"
 
     """
 }
 
 fontAwesomeEnum += """
-}
 
-/// An array of FontAwesome icon codes.
-// swiftlint:disable identifier_name
-public let FontAwesomeIcons: [String: String] = [
+    /// An unicode code of FontAwesome icon
+    public var unicode: String {
+        switch self {
 
 """
 
+
 sortedKeys.forEach { key in
     guard let value = icons[key] else { return }
+    let enumKeyName = key.filteredKeywords().camelCased(with: "-")
     fontAwesomeEnum += """
-        \"fa-\(key)\": \"\\u{\(value.unicode)}\"
+                case .\(enumKeyName): return \"\\u{\(value.unicode)}\"
     """
     if key != sortedKeys.last {
-        fontAwesomeEnum += ",\n"
+        fontAwesomeEnum += "\n"
     }
 }
 
 fontAwesomeEnum += """
+        
+        }
+    }
+}
 
-]
-
+"""
+/*
```

Comment:
Why is this commented out?

Reply:
I'd like to delete that part. But I don't know why it is there. The generated part is not used anywhere inside the framework.

Reply:
It was added via #202 and #203.

Reply:
It's part of the public interfaces and is used by the users of this framework.

Reply:
Okay, I didn't know that. So in that case I will update codegen to generate this enumeration as well.